src/libostree/ostree-autocleanups.h

```diff
@@ -59,6 +59,10 @@ G_DEFINE_AUTOPTR_CLEANUP_FUNC (OstreeSysrootUpgrader, g_object_unref)
 
 G_DEFINE_AUTO_CLEANUP_CLEAR_FUNC (OstreeRepoCommitTraverseIter, ostree_repo_commit_traverse_iter_clear)
 
+#ifdef OSTREE_ENABLE_EXPERIMENTAL_API
```

Comment:
There's something subtle here which I think is worth calling out; because this header file is public, this `#define` is used both for the build of ostree itself, and *also* clients will have to `#define` it.

That seems OK, but it'd probably be worth at least a comment in `configure.ac` that the name of the define is effectively API.

Or, we could use a different `#define` here for clients - `OSTREE_I_WANT_EXPERIMENTAL_API` ?  (And define it in our build if the automake conditional is set)